### ISSUE TITLE
Details page border color

### ISSUE THREAD

**Issue by melissa-lefevre** [department-of-veterans-affairs/va.gov-team#89147]
## Description
The border on the details page does not meet WCAG standards. Change border from #DFE1E2 to #757575

---

## Acceptance Criteria
- [x] Change border color to #757575

## Design artifacts
Link to[ figma file](https://www.figma.com/design/ktX3at2KixnYf2G6MoBSuC/MHV---Unified-Appointments-Library?node-id=482-1417&t=SZnvJFt3gHoXXVc0-1) with correct border color
<img width="341" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBelVHQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--61cddd7c1bc5ae191c5491e2de9299c7b672440e/Screenshot%202024-07-24%20at%2010.19.23%E2%80%AFAM.png" alt="Screenshot 2024-07-24 at 10.19.23 AM.png" />

## Definition of Done
- [ ] All acceptance criteria are met
- [ ] Documentation is updated

**Comment by Bren22va** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Appointments-Team-603fdef281af6500110a1691/issues/department-of-veterans-affairs/va.gov-team/89147?planning-poker) with Zenhub @cferris32 @jenniemc @JunTaoLuo @ryanshaw @simiadebowale @vbahinwillit

**Comment by ldelacosta** [comment]
@melissa-lefevre - ready to be validated in staging.

**Comment by outerpress** [comment]
@ldelacosta @jenniemc looks good! Validated

**Comment by ldelacosta** [comment]
No issues to report. Closing out the ticket.
